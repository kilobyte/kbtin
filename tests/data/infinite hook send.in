#hook send blah
#run a cat >/dev/null
x
#end
