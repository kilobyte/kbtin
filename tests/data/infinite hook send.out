#TOO DEEP RECURSION.

#SESSION 'a' DIED.
